lib/eventboss/error_handlers/db_connection_drop_handler.rb

```diff
@@ -0,0 +1,12 @@
+module Eventboss
+  module ErrorHandlers
+    class DbConnectionDropHandler
+      def call(exception, _context = {})
+        if exception.class == ::ActiveRecord::StatementInvalid
```

Comment:
Can we automatically add this handler if `ActiveRecord::StatementInvalid` is defined?